src/components/AuthForm.js

```diff
@@ -178,133 +185,147 @@ const loginPasswordUiSchema = {
+            required: ["username", "password"],
+            properties: {
+              username: {
+                type: "string",
+                title: "Email",
+                default: "jdoe@mozilla.com",
+              },
+              password: {
+                type: "string",
+                title: "Password",
+              },
             },
           },
         },
       },
+      uiSchema: {
+        ...baseUISchema,
+        credentials: {
+          password: { "ui:widget": "password" },
+        },
+      },
     },
-    uiSchema: {
-      ...baseUISchema,
-      credentials: {
-        password: { "ui:widget": "password" },
+    portier: {
+      schema: {
+        ...baseAuthSchema,
+        required: [...baseAuthSchema.required, "email"],
+        properties: {
+          ...baseAuthSchema.properties,
+          email: {
+            title: "Email address",
+            type: "string",
+            format: "email",
+          },
+        },
+      },
+      uiSchema: {
+        authType: {
+          ...baseUISchema.authType,
+          "ui:help": (
+            <span>
+              <b>Note:</b> The
+              <a href="https://github.com/Kinto/kinto-portier">
+                {" kinto-portier "}
+              </a>
+              plugin must be installed on the target server.
+            </span>
+          ),
+        },
       },
     },
-  },
-  portier: {
+  };
+  if (authType in customizedSchemas) {
+    return customizedSchemas[authType];
+  }
+
+  // Return the standard schemas if no customization is needed (eg for
+  // anonymous, openid).
+  return {
     schema: {
       ...baseAuthSchema,
-      required: [...baseAuthSchema.required, "email"],
-      properties: {
-        ...baseAuthSchema.properties,
-        email: {
-          title: "Email address",
-          type: "string",
-          format: "email",
-        },
-      },
     },
     uiSchema: {
-      authType: {
-        ...baseUISchema.authType,
-        "ui:help": (
-          <span>
-            <b>Note:</b> The
-            <a href="https://github.com/Kinto/kinto-portier">
-              {" kinto-portier "}
-            </a>
-            plugin must be installed on the target server.
-          </span>
-        ),
-      },
+      ...baseUISchema,
     },
-  },
+  };
 };
 
-const authLabels = {
-  anonymous: "Anonymous",
-  basicauth: "Basic Auth",
-  account: "Kinto Account Auth",
-  fxa: "Firefox Account",
-  ldap: "LDAP",
-  portier: "Portier",
+const authLabels = authType => {
+  const labels = {
+    anonymous: "Anonymous",
+    basicauth: "Basic Auth",
+    account: "Kinto Account Auth",
+    fxa: "Firefox Account",
+    ldap: "LDAP",
+    portier: "Portier",
+  };
+  if (authType.startsWith("openid-")) {
+    const provider = authType.replace("openid-", "");
+    const prettyProvider =
+      labels[provider] || provider.charAt(0).toUpperCase() + provider.slice(1);
+    return `${prettyProvider} (OpenID)`;
```

Comment:
If `authType` is something like `openid-Auth0.com` (aka `openid-${provider.name}`) then the name *is* `authType.replace("openid-", "");`

I think it should be the other way around. I.e.
```javascript
const providerName = authType.replace("openid-", "");
return `OpenID Connect (${provider})`
```

*Note!* It's called "OpenID Connect", not "OpenID". "OpenID" was some other junk we tried and failed 10-15 years ago. "OpenID Connect" is a OAuth2 sub-standard.